### PR TITLE
Fixed issue #9032

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
Fixed issue #9032

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 
Checked and Found while in signup process when i selected the Newsletter subscription, still it is not selected in profile. after editing and saving the profile, then it is updated.

**Video**


https://github.com/bagisto/bagisto/assets/92721837/3dbf92a3-c042-4aef-8cb2-b189375f14a9

**Comment by shivendra-webkul** [comment]
@ashishkumar-webkul Fixed for sign up page please check.

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 
as i have checked and found that the issue has been fixed from the signup page as well. Please check  the video for the reference.

**Video**


https://github.com/bagisto/bagisto/assets/92721837/13e6c03f-bfff-4117-8831-be432a2a5ad9

File: packages/Webkul/Shop/src/Http/Controllers/Customer/RegistrationController.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public function store(RegistrationRequest $registrationRequest)
             'is_verified'               => ! core()->getConfigData('customer.settings.email.verification'),
             'customer_group_id'         => $this->customerGroupRepository->findOneWhere(['code' => 'general'])->id,
             'token'                     => md5(uniqid(rand(), true)),
-            'subscribed_to_news_letter' => request()->input('is_subscribed') ?? 0,
+            'subscribed_to_news_letter' => request()->input('is_subscribed') ? '1' : '0',
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Http/Controllers/Customer/RegistrationController.php:60]: The following will work too, don't use unnecessary ternary operator

(boolean) request()->input('is_subscribed')

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**